packages/core/src/cli/add-connector.ts

```diff
@@ -0,0 +1,25 @@
+import 'module-alias/register';
+import { getEnv } from '@silverhand/essentials';
+import chalk from 'chalk';
+
+import { addConnector } from '@/connectors/add-connectors';
+import { defaultConnectorDirectory } from '@/env-set';
+
+import { configDotEnv } from '../env-set/dot-env';
+
+configDotEnv();
+
+const addConnectorCli = async (packageName: string) => {
```

Comment:
we can automatically fulfill the package name like eslint:

```
"gmail" -> "@logto/connector-gmail" fall back to "logto-connector-gmail"
"@foo/discord" -> "@foo/logto-connector-discord"
```

Reply:
also multiple connectors support?

Reply:
LOG-4052